requirements.txt

```
@@ -1,4 +1,4 @@
-ensembl-hive@git+https://github.com/Ensembl/ensembl-hive.git@master#egg=ensembl-hive
+ensembl-hive@git+https://github.com/Ensembl/ensembl-hive.git@main#egg=ensembl-hive
```

URL Fragment `egg` is redundant as `ensembl-hive` has a `setup.py` that sets the project name.

```suggestion
ensembl-hive@git+https://github.com/Ensembl/ensembl-hive.git@main
```